Formula/pixiewps.rb

```diff
@@ -0,0 +1,20 @@
+class Pixiewps < Formula
+  desc "Offline Wi-Fi Protected Setup brute-force utility"
+  homepage "https://github.com/wiire-a/pixiewps"
+  url "https://github.com/wiire-a/pixiewps/releases/download/v1.4.2/pixiewps-1.4.2.tar.xz"
+  sha256 "c4dc0569e476ebdbd85992da2d1ff799db97ed0040da9dc44e13d08a97a9de1e"
+  license "GPL-3.0-or-later"
+  head "https://github.com/wiire-a/pixiewps.git", branch: "master"
+
+  depends_on "openssl@3"
+
+  def install
+    inreplace "Makefile", "/usr/local", prefix.to_s
+
+    system "make", "OPENSSL=1", "install"
+  end
+
+  test do
+    assert_match version.to_s, shell_output("#{bin}/pixiewps --version 2>/dev/null", 3)
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula